users/migrations/0027_set_username_to_email.py

```diff
@@ -0,0 +1,30 @@
+from django.db import migrations
+
+
+def update_usernames_to_email(apps, schema_editor):
+    """Update usernames to match user email addresses"""
+    User = apps.get_model("users", "User")
+
+    # Update usernames to match emails but truncate at max length
+    # Get max_length from username field
+    max_length = User._meta.get_field("username").max_length  # noqa: SLF001
```

Comment:
I forgot this column had a lower max_length - if we have to truncate the data there isn't much point in doing this work anyway - so can we widen the column to match `email` here too so you don't have to truncate it?

Reply:
That should be plenty, looks like there's a default of 254 for `EmailField` in django based on specs: https://github.com/django/django/blob/stable/5.2.x/django/db/models/fields/__init__.py#L1923-L1924